### PR TITLE
Gcp restart and list database instances operation and workflow

### PR DESCRIPTION
**Pull request by KalyankarRajesh**
Following operations and workflow created

List Database Instance 
Restart Database Instance
Restart Database Instance Workflow

### CODE REVIEW

**Review by umavanth21 — APPROVED** [review]

File: content/io/cloudslang/google/databases/instances/list_database_instances.sl [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+#!                        Default value: '..JAVA_HOME/java/lib/security/cacerts'
+#!                        Format: Java KeyStore (JKS)
+#!                        Optional
+#! @input trust_password: The password associated with the trust_keystore file. If trust_all_roots is false
+#!                        and trust_keystore is empty, trust_password default will be supplied.
+#!                        Optional
+#!
+#! @output return_result: This will contain the response entity.
+#! @output status_code: 200 if request completed successfully, others in case something went wrong.
+#! @output instances_json: A JSON list containing the Instances information.
```
Comment (patmante) [inline content/io/cloudslang/google/databases/instances/list_database_instances.sl]: change the output name and description

File: content/io/cloudslang/google/databases/instances/list_database_instances.sl [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+#! @output list_database_instance_names: List of database instance names.
+#!
+#! @result SUCCESS: The database instance details successfully retrieved.
```
Comment (patmante) [inline content/io/cloudslang/google/databases/instances/list_database_instances.sl]: update the description

File: content/io/cloudslang/google/databases/instances/restart_database_instance.sl [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+#!                        Optional
+#! @input trust_password: The password associated with the trust_keystore file. If trust_all_roots is false
+#!                        and trust_keystore is empty, trust_password default will be supplied.
+#!                        Optional
+#!
+#! @output return_result: This will contain the response entity.
+#! @output status_code: 200 if request completed successfully, others in case something went wrong.
+#! @output database_instance_json: A JSON list containing the database instances information.
+#!
+#! @result SUCCESS: The database instance details successfully retrieved.
```
Comment (patmante) [inline content/io/cloudslang/google/databases/instances/restart_database_instance.sl]: update the description

**Review by patmante — CHANGES_REQUESTED** [review]

File: content/io/cloudslang/google/databases/instances/list_database_instances.sl (hunk 1 above)
Comment (KalyankarRajesh) [reply content/io/cloudslang/google/databases/instances/list_database_instances.sl]: updated

File: content/io/cloudslang/google/databases/instances/list_database_instances.sl (hunk 2 above)
Comment (KalyankarRajesh) [reply content/io/cloudslang/google/databases/instances/list_database_instances.sl]: updated

File: content/io/cloudslang/google/databases/instances/restart_database_instance.sl (hunk 3 above)
Comment (KalyankarRajesh) [reply content/io/cloudslang/google/databases/instances/restart_database_instance.sl]: updated

**Review by patmante — APPROVED** [review]

**Pull request merged**